domain/user/service/service.go

```diff
@@ -345,6 +355,38 @@ func (s *Service) ResetPassword(ctx context.Context, uuid user.UUID) ([]byte, er
 	return activationKey, nil
 }
 
+// EnableUser will enable the user for authentication.
+//
+// The following error types are possible from this function:
+// - usererrors.UUIDNotValid: When the UUID supplied is not valid.
+// - usererrors.NotFound: If no user by the given UUID exists.
+func (s *Service) EnableUser(ctx context.Context, uuid user.UUID) error {
```

Comment:
I think we should get a name change here to `EnableUserAuthentication`. That way it is clear what we are controlling. I think it could be very easy for people to confuse concepts like removed and disabled if we don't qualify it a bit more.

Reply:
agreed